internal/slicer/slicer.go

```diff
@@ -170,9 +172,18 @@ func Run(options *RunOptions) (*Report, error) {
 				if extractInfo == nil {
 					return nil
 				}
-				if _, ok := slice.Contents[extractInfo.Path]; !ok {
+				if _, ok := pathInfos[extractInfo.Path]; !ok {
```

Comment:
What does that verification actually tell us?

Reply:
It tells us whether the path is in any slice that belong to the selection (precisely the contents of pathInfo). In future PRs we use this to know if the path is `until: `.

Right now, it could be replaced with a boolean in the loop below, but I think this is clearer and we use it in the future, but happy to change it.